edge-modules/azure-monitor/src/README.md

```diff
@@ -0,0 +1,126 @@
+# Azure IoT Edge Metrics Collector
+
+## About
+This repository contains souce code for Azure IoT Edge Metrics Collector Hub devices. It is currently in a private preview process.
```

Comment:
> This repository contains souce code for Azure IoT Edge Metrics Collector Hub devices.

This repository contains source code for the Azure IoT Edge Metrics module.

Reply:
Changed.